### DISCUSSION TITLE
Dynamic Report Parameters

### DISCUSSION THREAD

**Discussion by flashyjunior** [serenity-is/Serenity#7228] [Help]
I have a report that I inherit from `IReport, ICustomizeHtmlToPdf`
I have 2 parameters as seen below.
Based on the **Event** option selected, I want to load another lookupeditor and show below the Event lookupeditor on the report dialog for user selection before generating the report.
Please any help will be appreciated

![image](https://github.com/user-attachments/assets/5ecd99fd-5387-43c8-8bc5-fc5d310b2767)

**Comment by L30P1N0** [comment]
Could you create a cascading combo box that, based on the user's selection in the EVENT, shows or hides options depending on the cascade?

Then, when triggering the report, you should validate the values obtained from the lookupEditor to display the report with the parameters correctly set based on the lookupEditors.

**Comment by flashyjunior** [comment]
Thank you @L30P1N0 for you help.
I want to load the lookupeditor dynamically when the `ReportDialog` shows up, based on the reportkey option value

**Comment by flashyjunior** [comment]
Hi,
Please any assistance.
Thank you @L30P1N0
